mail-client/neomutt/neomutt-20200821.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils flag-o-matic
```

Comment:
When I skimmed, I wasn't sure if we were using flag-o-matic anymore

Reply:
done